components/style/properties/longhand/box.mako.rs

```diff
@@ -1149,112 +1149,64 @@ ${helpers.predefined_type("scroll-snap-coordinate",
-                m33: Number::from_computed_value(&computed.m33),
-                m34: Number::from_computed_value(&computed.m34),
-                m41: Number::from_computed_value(&computed.m41),
-                m42: Number::from_computed_value(&computed.m42),
-                m43: Number::from_computed_value(&computed.m43),
-                m44: Number::from_computed_value(&computed.m44),
-            }
-        }
-    }
-
-    fn parse_two_lengths_or_percentages(context: &ParserContext, input: &mut Parser)
-                                        -> Result<(specified::LengthOrPercentage,
-                                                   specified::LengthOrPercentage),()> {
-        let first = try!(specified::LengthOrPercentage::parse(context, input));
-        let second = input.try(|input| {
-            try!(input.expect_comma());
-            specified::LengthOrPercentage::parse(context, input)
-        }).unwrap_or(specified::LengthOrPercentage::zero());
-        Ok((first, second))
-    }
-
-    fn parse_two_numbers(input: &mut Parser) -> Result<(Number, Number), ()> {
-        let first = try!(specified::parse_number(input));
-        let second = input.try(|input| {
-            try!(input.expect_comma());
-            specified::parse_number(input)
-        }).unwrap_or(first);
-        Ok((first, second))
-    }
-
-    fn parse_two_angles(context: &ParserContext, input: &mut Parser)
-                       -> Result<(specified::Angle, specified::Angle),()> {
-        let first = try!(specified::Angle::parse(context, input));
-        let second = input.try(|input| {
-            try!(input.expect_comma());
-            specified::Angle::parse(context, input)
-        }).unwrap_or(specified::Angle::zero());
-        Ok((first, second))
-    }
-
-    #[derive(Copy, Clone, Debug, PartialEq)]
-    #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
-    enum TranslateKind {
-        Translate,
-        TranslateX,
-        TranslateY,
-        TranslateZ,
-        Translate3D,
-    }
-
+    /// Describes a single parsed
+    /// [Transform Functions](https://drafts.csswg.org/css-transforms/#typedef-transform-function).
+    ///
+    /// Multiple transform functions compose a transformation.
+    ///
+    /// Some transformations can be expressed by other more general functions.
     #[derive(Clone, Debug, PartialEq)]
     #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
-    enum SpecifiedOperation {
-        Matrix(SpecifiedMatrix),
-        Skew(specified::Angle, specified::Angle),
-        Translate(TranslateKind,
-                  specified::LengthOrPercentage,
-                  specified::LengthOrPercentage,
-                  specified::Length),
-        Scale(Number, Number, Number),
-        Rotate(Number, Number, Number, specified::Angle),
+    pub enum SpecifiedOperation {
+        /// Represents a 2D 2x3 matrix.
+        Matrix { a: Number, b: Number, c: Number, d: Number, e: Number, f: Number },
```

Comment:
The spec already calls the matrix members a, b, c, d, e, f:
https://drafts.csswg.org/css-transforms/#MatrixDefined